src/main.py

```diff
@@ -11,6 +11,9 @@
 
 pg.init()  # pylint: disable=no-member
 
+#Variable
```

Comment:
- Please leave a whitespace after `#`
- Useless comment:
Either create a useful command or do not command